src/ir_LG.h

```diff
@@ -1,8 +1,13 @@
 // Copyright 2017, 2019 David Conran
 
+/// @file
+/// @brief Support for LG protocols.
+/// @see https://github.com/arendst/Tasmota/blob/54c2eb283a02e4287640a4595e506bc6eadbd7f2/sonoff/xdrv_05_irremote.ino#L327-438
+
+
 // Supports:
-//   Brand: LG,  Model: 6711A20083V remote
-//   Brand: LG,  Model: AKB74395308 remote
+//   Brand: LG,  Model: 6711A20083V remote (LG)
+//   Brand: LG,  Model: AKB74395308 remote (LG)
```

Comment:
removed comments mixes AKB74395308 between `decodeLG` and `sendLG2` ?

Reply:
Corrected to LG2